server/mlabns/handlers/lookup.py

```diff
@@ -12,54 +13,17 @@
 from google.appengine.ext.webapp import template
 
 
-def _create_af_aware_fqdn(fqdn, address_family):
-    """Adds the v4/v6 only annotation to the fqdn.
-
-    Example:
-        fqdn:       'npad.iupui.mlab3.ath01.measurement-lab.org'
-        ipv4 only:  'npad.iupui.mlab3v4.ath01.measurement-lab.org'
-        ipv6 only:  'npad.iupui.mlab3v6.ath01.measurement-lab.org'
-
-    Args:
-        fqdn: A tool FQDN with no address family specific annotation.
-        address_family: The address family for which to create the FQDN or None
-            to create an address family agnostic FQDN.
-
-    Returns:
-        A FQDN specific to a particular address family, or the original FQDN
-        if an address family is not specified.
-    """
-    if address_family == message.ADDRESS_FAMILY_IPv4:
-        fqdn_annotation = 'v4'
-    elif address_family == message.ADDRESS_FAMILY_IPv6:
-        fqdn_annotation = 'v6'
-    elif not address_family:
-        fqdn_annotation = ''
-    else:
-        logging.error('Unrecognized address family: %s', address_family)
-        return fqdn
-
-    fqdn_parts = fqdn.split('.')
-    fqdn_parts[2] += fqdn_annotation
-
-    return '.'.join(fqdn_parts)
-
-
-def _create_tool_url(fqdn, address_family, http_port):
+def _create_tool_url(fqdn, http_port):
     """Create an HTTP URL for a tool.
 
     Args:
-        fqdn: A tool FQDN with no address family specific annotation.
-        address_family: The address family for which to create the URL or None
-            to create an address family agnostic URL.
+        fqdn: A tool FQDN (already processed by fqdn_rewrite).
         http_port: The HTTP port number part of the URL.
 
     Returns:
-        A tool URL, annotated with the appropriate address family (when
-        available).
+        A constructed tool HTTP URL with protocol and http port.
     """
-    return 'http://%s:%s' % (_create_af_aware_fqdn(fqdn, address_family),
-                             http_port)
+    return 'http://%s:%s' % (fqdn, http_port)
```

Comment:
> Also, it should be noted, for example, that the following URL does give you a secure connection in a browser, but just doesn't run a test properly:

>https://ndt-iupui-mlab4-nuq1t.measurement-lab.org:3001/

Right this is expected. The browser will connect to 3001 and do a TLS handshake, then try to speak HTTP. The server will accept a TLS handshake, then try to speak NDT, so you get undefined behavior because they're talking to each other over different protocols. I just tried this with Chrome on Ubuntu and the browser thinks the server is serving a small binary file and downloads it.

>If we don't care about advertising the SSL port in any way, then I guess the best thing to do is simply leave the non-SSL fakewww URL, despite the fact that it's a little confusing to return non-SSL resources for a specifically SSL request.

I respectfully disagree. If we have a client that requests a web portal for `/ndt-ssl` it doesn't make sense to direct them to a web portal for ndt plaintext. We don't have a web portal for NDT-SSL, and so I think it's accurate for us to omit the `http_port` in the response for `/ndt-ssl` because omitting that means "we don't have a web interface for this server."